controllers/admin/AdminBlockListingController.php

```diff
@@ -140,7 +140,12 @@ public function displayAjaxSaveBlockContent()
             $filename = $customImage['name'];
 
             // validateUpload return false if no error (false -> OK)
-            $validUpload = ImageManager::validateUpload($customImage);
+            $validUpload = ImageManager::validateUpload(
+                $customImage,
+                0,
+                ['gif', 'jpg', 'jpeg', 'jpe', 'png', 'svg'],
+                ['image/gif', 'image/jpg', 'image/jpeg', 'image/pjpeg', 'image/png', 'image/x-png', 'image/svg']
```

Comment:
if it needs a change on the develop branch, you must update compatibility to 1.7.7.0 :)

Reply:
Fixed :heavy_check_mark: